### PR TITLE
Consul on docker.

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

File: roles/consul/tasks/config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# tasks file for consul
+- name: remove consul override
```
Comment (tayzlor) [inline roles/consul/tasks/config.yml]: is this still needed?

File: roles/consul/tasks/config.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  file:
+    path: /etc/init/consul.override
+    state: absent
+
+- service: name=consul state=stopped enabled=no
```
Comment (tayzlor) [inline roles/consul/tasks/config.yml]: needed?

**Comment by wallies** [comment]
@enxebre this might be worth another go with the new version on consul

**Comment by tayzlor** [comment]
Closing in favour of #585

**Pull request closed without merging**